lib/docker.ts

```diff
@@ -91,6 +101,25 @@ class Docker {
     ]);
   }
 
+  public runAsStream(
+    cmd: string,
+    args: string[] = [],
+    cb: subProcess.ExecuteAsStreamCallback,
+  ) {
+    return subProcess.executeAsStream("docker", cb, [
+      ...this.optionsList,
+      "run",
+      "--rm",
```

Comment:
I assume we don't want that, we will delete the docker image after we finish :)
even worse, we will re-pull the image on every call to `runAsStream`

Reply:
don't mind me resolving this conversation please, ```--rm``` tells Docker to delete the container, not the image, after running, so it's actually cleaner and we like it ❤️